### PR TITLE
track the project id a user signs up for

### LINKED ISSUE #901: display_name should always be treated as case insensitive
We should store it with the supplied case but for searches and logins we should ignore the user name's case.

### PR DESCRIPTION
**Pull request by camallen**
closes #901 allow the project_id param to track which project a user is signing up for when registering.

### CODE REVIEW

**Comment by camallen** [comment]
@mrniaboc any desire to know which project a user signed up for after the fact? If so i'll have to persist this in the database.

**Comment by mrniaboc** [comment]
Yes, I think that will be very useful to know.

File: app/models/user.rb [hunk 1]
```diff
@@ -2,6 +2,9 @@ class User < ActiveRecord::Base
   include Activatable
   include Linkable
 
+  #on registration track which project the user signs up with
+  attr_accessor :project_id
```
Comment (edpaget) [inline app/models/user.rb]: Are you not persisting this anywhere?

Comment (camallen) [inline app/models/user.rb]: nope, was just going to pass it to the welcome emailer to modify the content there. I'll add a field to the users table to track it.

Comment (edpaget) [inline app/models/user.rb]: :+1:

**Pull request merged**